### ISSUE TITLE
Custom airlines and aircraft

### ISSUE THREAD

**Issue by net47** [johanohly/AirTrail#158]
### The airport(s)

TXL

### Additional information

Would be great if someone can add custom airports. Useful when airports are already closed, but off course should be used in flights of past years. 

It would also be great if someone could add custom airlines which again don't exist anymore - like Air Berlin (http://ab-inso.de/index-en.html), GermanWings (https://en.wikipedia.org/wiki/Germanwings) or FlyNiki (http://www.flyniki.com/index-en.html).

**Comment by nocomment-bln** [comment]
I also can not find airports. All Airports in Capo Verde are missing.

**Comment by nocomment-bln** [comment]
Would also like to have this feature :) i am also from Berlin

**Comment by johanohly** [comment]
A new airport data source is being used in v2.0.0, which has records of closed airports as well. 

Along with that, the application now allows the addition of custom airports!

I'll leave this issue open to track the interest for custom airlines (and aircraft).

**Comment by caracturus** [comment]
Custom aircraft would be fantastic!

**Comment by Drone8334** [comment]
I would definitely appreciate custom aircraft, as I want to use this to log primarily general aviation flights.

**Comment by planes-n-crashes** [comment]
Same here, I‘d love to use it for my own GA flights and add just the airplane type on the fly while adding the flight.
Same with the flight number to be able to distinguish my own flying from airline flying.
Thanks for considering it.

**Comment by avee87** [comment]
About TXL missing: apparently, closed airports cannot have ICAO, IATA and GPS codes on ourairports.com. So AirTrail doesn't have these codes and is unable to match flights to closed airports.

**Comment by johanohly** [comment]
> About TXL missing: apparently, closed airports cannot have ICAO, IATA and GPS codes on ourairports.com. So AirTrail doesn't have these codes and is unable to match flights to closed airports.

Yeah, I was able to find TXL by name. I honestly didn't think about the fact that imports wont be able to match them without the code. 
Curiously, TXL does still have the ICAO code in AirTrail, but Schönefeld doesn't. I assume it was transferred to Brandenburg, while TXL didn't need to transfer its' code because no airport replaced it.

**Comment by net47** [comment]
Version 2.0.0 now supports custom airports
